mLRS/CommonRx/mlrs-rx.cpp

```diff
@@ -774,18 +792,55 @@ dbg.puts(s8toBCD_s(stats.last_rssi2));*/
             msp.FrameLost();
         }
 
+#if 0
+stats.received_fhss_index = 63; // enable/disable fhss_index mechanism
+#endif
+
+        // check fhss index
+        // do it only in LISTEN and SYNC
+        if ((connect_state <= CONNECT_STATE_SYNC) && valid_frame_received &&
+            (stats.received_fhss_index < 63)) { // older version don't offer this
+dbg.puts(" check ");
+            if (stats.received_fhss_index_band == 0) {
+                if (fhss.GetCurrI() != stats.received_fhss_index) { // somehow wrong frequency, discard
+                    valid_frame_received = false;
+                    invalid_frame_received = true;
+dbg.puts(" discard");
+                    fhss.SetCurrI(stats.received_fhss_index); // set band 1, should only happen for dual band
+dbg.puts(" set b1");
+                } else { // ok
+                    connect_fhss_index_band_seen |= 0x01; // 0 seen
+                }
+            } else {
+                if (fhss.GetCurrI2() != stats.received_fhss_index) { // somehow wrong frequency, so discard
+                    valid_frame_received = false;
+                    invalid_frame_received = true;
+dbg.puts(" discard");
+                    fhss.SetCurrI2(stats.received_fhss_index); // set band 2, should only happen for dual band
```

Comment:
I would prefer the comment to say "set second band index"...  ie received_fhs_index_band == 1 is second band, not band 2.  Same thing above, I would prefer comment say "first band" rather than "band 1" when received_fhs_index_band = 0.

Reply:
THX. done.